### PR TITLE
refactor(composable)!: refactor useBilling

### PR DESCRIPTION
**Pull request by bartoszherba**
## Description
refactor useBilling

## Related Issue
-

## Motivation and Context
refactoring

## How Has This Been Tested?
-

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/composables/useBilling/commands/saveBillingAddressCommand.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    };
+
+    const { data } = await context.$vsf.$magento.api.setBillingAddressOnCart(
+      setBillingAddressOnCartInput,
+    );
+
+    Logger.debug('[Result]:', { data });
+
+    /**
+     * End of GraphQL Workaround
```
Comment (Frodigo) [inline packages/theme/composables/useBilling/commands/saveBillingAddressCommand.ts:38]: What workaround do you mean?

File: packages/theme/helpers/userAddressManipulator.ts [hunk 2]
```diff
@@ -1,78 +0,0 @@
-import { CustomerAddressInput } from '~/modules/GraphQL/types';
```
Comment (Frodigo) [inline packages/theme/helpers/userAddressManipulator.ts:1]: Why did you delete this file? Other composables usu it as well. Can you restore this change?

**Review by Frodigo — CHANGES_REQUESTED** [review]

File: packages/theme/composables/useBilling/commands/saveBillingAddressCommand.ts (hunk 1 above)
Comment (bartoszherba) [reply packages/theme/composables/useBilling/commands/saveBillingAddressCommand.ts:38]: The one in index.ts ;) i moved it.

File: packages/theme/helpers/userAddressManipulator.ts (hunk 2 above)
Comment (bartoszherba) [reply packages/theme/helpers/userAddressManipulator.ts:1]: Ok.

**Review by Frodigo — APPROVED** [review]
